src/Rules/StarterWeb/AI/IndividualAuth/project.json

```diff
@@ -53,8 +54,7 @@
     "Microsoft.EntityFrameworkCore.Tools": {
       "version": "__ToolsPackageVersion__",
       "imports": [
-        "portable-net45+win8+dnxcore50",
-        "portable-net45+win8"
+        "portable-net451+win8"
```

Comment:
In making this change, I am assuming https://github.com/dotnet/cli/pull/3386 will be taken in which bumps to the netstandard version of JSON.NET